app.js

```diff
@@ -101,28 +102,35 @@ function generateNewImages(){
 
 }
 
-//functoin to track total number of clicks per picture and call the functoin to generate a new set of pictures
+//functoin to track total number of clicks per picture and call the function to generate a new set of pictures
 function countClicks(event) {
 
   totalNumberOfClicks++;
   for(var i = 0; i < pictureArray.length; i++){
     if(event.target.name === pictureArray[i].name)
       pictureArray[i].numberOfTimesClicked++;
-      // console.log(pictureArray[i].numberOfTimesClicked);
+    pictureArray[i].percentage = Math.round(pictureArray[i].numberOfTimesClicked / pictureArray[i].numberOfTimesShowned * 100);
```

Comment:
Look into adding a break statement here.   this will cut out of the loop early if it finds a match, otherwise it will continue to loop through all remaining items.